packages/mobile/src/components/lineup-tile/TrackTileMetrics.tsx

```diff
@@ -24,17 +24,18 @@ import { VanityMetric } from './VanityMetrics'
 
 const { setFavorite } = favoritesUserListActions
 const { setRepost } = repostsUserListActions
-
+const { getTrack } = cacheTracksSelectors
 type RepostsMetricProps = {
   trackId: ID
 }
 
 export const RepostsMetric = (props: RepostsMetricProps) => {
   const { trackId } = props
-  const { data: track } = useGetTrackById(
-    { id: trackId },
-    { disabled: !trackId }
-  )
+
+  const repostCount = useSelector((state: CommonState) => {
```

Comment:
same for these, why not either destructure from collection, or pull out to shared selector?